cmudb/exploration/exploratory_daemon.py

```diff
@@ -0,0 +1,132 @@
+import argparse
+import subprocess
+from typing import Tuple
+
+from data_copy import copy_pgdata_cow, destroy_exploratory_data_cow
+from pgnp_docker import start_exploration_docker, shutdown_exploratory_docker, setup_docker_env
+from sql import checkpoint, execute_sql, \
+    wait_for_pg_ready
+from util import ZFS_DOCKER_VOLUME_POOL, REPLICA_VOLUME_POOL, REPLICA_PORT, EXPLORATION_PORT, \
+    EXPLORATION_CONTAINER_NAME, \
+    DOCKER_VOLUME_DIR, execute_sys_command
+
+
+def main():
+    """
+    The exploratory daemon is responsible for creating a copy of replica instances, to be used for model training.
```

Comment:
@lmwnshn Is this clear enough or too vague? I didn't want to dump a bunch of commands to run but I also wanted to provide some detail.